docs/index.md

```diff
@@ -49,13 +48,15 @@ development version are subject to change at any time before the next stable rel
 
 ### Install on an offline machine
 
-To install TSC onto a machine without internet connection, use the following steps:
+To install TSC onto a machine without an internet connection, use the following steps:
+
+1) Ensure that Python 3.6 or higher is installed.
```

Comment:
I think we should say 3.5 or higher here as well for consistency

Reply:
Ack